src/System.Management.Automation/namespaces/LocationGlobber.cs

```diff
@@ -3339,8 +3339,10 @@ internal static string GetDriveQualifiedPath(string path, PSDriveInfo drive)
                 }
                 else
                 {
-                    if (path.StartsWith(StringLiterals.DefaultPathSeparatorString, StringComparison.Ordinal) || 
-                        path.StartsWith(StringLiterals.AlternatePathSeparatorString, StringComparison.Ordinal)) 
+                    if ((path.StartsWith(StringLiterals.DefaultPathSeparatorString, StringComparison.Ordinal) || 
```

Comment:
Please, add more comments explaining what you are doing here. It will be hard to understand the original intention if you look at this code again after some time. This is required by [Code comments](https://github.com/PowerShell/PowerShell/blob/master/docs/dev-process/coding-guidelines.md#code-comments).